ui/app/mainui/AppMain.qml

```diff
@@ -1117,7 +1117,8 @@ Item {
                                     openCreateChat: createChatView.opened
                                     chatCommunitySectionModule: {
                                         appMain.rootStore.mainModuleInst.prepareCommunitySectionModuleForCommunityId(model.id)
-                                        return appMain.rootStore.mainModuleInst.getCommunitySectionModule()
+                                        appMain.rootStore.chatCommunitySectionModule = appMain.rootStore.mainModuleInst.getCommunitySectionModule()
+                                        return appMain.rootStore.chatCommunitySectionModule
```

Comment:
I'm pretty sure that's what caused the issue I found. I think you can revert this now.

Reply:
done, removed